universum/lib/gravity.py

```diff
@@ -39,69 +43,80 @@ def __setattr__(self, key, value):
 
 
 class Settings:
-    def __init__(self, cls):
-        self.cls = cls
+    def __init__(self, cls: Type['Module']) -> None:
+        self.cls: Type['Module'] = cls
 
-    def __get__(self, obj, objtype=None):
+    def __get__(self, obj: 'Module', objtype: Any = None) -> 'ModuleSettings':
```

Comment:
I am wondering, would it make more sense to make it explicit that ModuleSettings is just a proxy object for ModuleNamespace by using corresponding annotation?

Reply:
hm, sounds interesting, but I don't think there is an easy way to do this. Let me think about it

Reply:
no longer seems to make sense, unless we want to closely couple ModuleNamespace and gravity